science/lal/Portfile

```diff
@@ -3,16 +3,17 @@
 PortSystem    1.0
 
 name          lal
-version       6.18.0
-revision      8
+version       6.19.0
+revision      1
```

Comment:
Remove the `revision` line because it starts at 0 when increasing the version and that is the default value.